### ISSUE TITLE
Case sensitive geometry column name comparison causes exception

### ISSUE THREAD

**Issue by asammartin** [ngageoint/geopackage-java#42]
Please fill out as much known and relevant information as possible.

#### Version Information:

  * GeoPackage Java Version: 3.0.2
  * GeoPackage Java Source: mvn central repo
  * IDE Name & Version: Intellij 2018.2
  * Maven Version: (mvn -version) n/a
  * Java Version: (java -version) 1.8.0_161
  * Platform & OS: MacOS 10.13.6
  * Other Relevant Libraries:
    * Gradle 4.7

#### Expected Results:

  * What did you expect to happen?
    * Case insensitive comparison of column names

#### Observed Results:

  * What happened instead?
  * Difference in geometry column name definition of "shape" and usage of "Shape" causes column not to be recognized as a geometry column and an exception thrown.
  * How often does this occur?
    * Every time geopackage with a geometry column name case mismatch is loaded

#### Output:

  * Any logs, errors, or output messages?
Error importing table rad_impwtmdls_a
java.lang.IllegalArgumentException: No enum constant mil.nga.geopackage.db.GeoPackageDataType.MULTIPOLYGON
	at java.lang.Enum.valueOf(Enum.java:238)
	at mil.nga.geopackage.db.GeoPackageDataType.valueOf(GeoPackageDataType.java:11)
	at mil.nga.geopackage.db.GeoPackageDataType.fromName(GeoPackageDataType.java:130)
	at mil.nga.geopackage.features.user.FeatureTableReader.createColumn(FeatureTableReader.java:60)
	at mil.nga.geopackage.features.user.FeatureTableReader.createColumn(FeatureTableReader.java:15)
	at mil.nga.geopackage.user.UserCoreTableReader.readTable(UserCoreTableReader.java:160)

#### Steps to Reproduce:

  1. Download test file (see link below)
  2. Load the gpkg
  3. Attempt to get dao for one of the feature tables

  1. Open test file in a db viewer
  2. View table `gpkg_geometry_columns` 
  3. Note the `column_name` values are of `shape`
  4. View the table `rad_303d_p`
  5. Note the name of the geometry column is `Shape`

#### Relevant Code:

```java
String tableName = "rad_303d_p";
geoPackage.getFeatureDao(tableName);
```

#### Test Files:

  * Links to any files needed for testing?
    * http://watersgeo.epa.gov/GEOSPATIALDOWNLOADS/rad_303d_20150501_gpkg.zip

#### Additional Information:

  * Any additional configuration, data, or information that might help with the issue?

**Comment by bosborn** [comment]
Made [this change](https://github.com/ngageoint/geopackage-java/commit/a58ae2213aa3476fdffe82fd91ef86fe1210de7f) for case insensitive comparison of the geometry column name.  Was able to iterate through the feature rows on your test GeoPackage.  Will release as part of 3.0.3+.

If you want to fix that GeoPackage using the current version, you can run the following:
```geoPackage.execSQL("update gpkg_geometry_columns set column_name = 'Shape' where column_name = 'shape'");```

Let me know if you want dev builds of the 3.0.3 jars.

**Comment by asammartin** [comment]
that change looks great. don't need the dev builds. have a temporary adapter to work around issues until 3.0.3. thanks.

**Comment by bosborn** [comment]
Fixed in [3.1.0](https://github.com/ngageoint/geopackage-java/releases/tag/3.1.0)
